logbook-core/src/main/java/org/zalando/logbook/CookieHeaderFilter.java

```diff
@@ -0,0 +1,102 @@
+package org.zalando.logbook;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Predicate;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.compile;
+import static java.util.stream.Collectors.toList;
+
+final class CookieHeaderFilter implements HeaderFilter {
+
+    @FunctionalInterface
+    private interface Processor {
+        void process(Matcher matcher, StringBuffer result);
+    }
+
+    private final Pattern pattern = compile("(?<name>[^=; ]+)=(?<value>[^; ]*)");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/whiskeysierra/logbook/pullRequest?prid=4979345)